modules/packages/freebsd_ports

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh -e
+
+command=$1
+
+while read -r line; do
+  export INPUT_$line
+done
+
+list_updates() {
+  pkg version -v -l "<" | sed -Ee $'s/([^ ]+)-[^-]+ .* (.*)\\)/Name=\\1\\\nVersion=\\2\\\nArchitecture=freebsd:10:x86:64/'
```

Comment:
Neither of those commands give me what we need here though - the latest version number available via ports:

```
$ pkg version -v -l "<" | grep weechat
weechat-1.0.1_1                    <   needs updating (index has 1.1.1_1)

$ pkg query '%n %v %q %R' | grep weechat
weechat 1.0.1_1 freebsd:9:x86:64 FreeBSD

$ pkg shell
SQLite version 3.8.8.2 2015-01-30 14:30:45
Enter ".help" for usage hints.
sqlite> select * from packages where name = 'weechat';

90|irc/weechat|weechat|1.0.1_1|Lightweight and user friendly ncurses based IRC client|WeeChat (Wee Enhanced Environment for Chat) is a fast and light IRC
client. Everything can be done with a keyboard, and it is customizable
and extensible with scripts.

WWW: http://weechat.org|||freebsd:9:x86:64|jase@FreeBSD.org|http://weechat.org|/usr/local|18505968|0|0|1||1417486125||2$0$tqhgmq53qqc6pzmd7f8oxx4f318zedk34y1idrtg4ycyyitniimb
```